tests/integration/test_core.py

```diff
@@ -89,9 +91,64 @@ async def test_mysql_relation(
     await ops_test.model.deploy("charmed-osm-mariadb-k8s", application_name="mariadb")
     await ops_test.model.add_relation("wordpress", "mariadb:mysql")
     await ops_test.model.wait_for_idle()
+
     assert (
             await get_app_status("wordpress") == ops.model.ActiveStatus.name
     ), (
         "application status should be active once correct database connection info "
         "being provided via relation"
     )
+
+
+@pytest.mark.asyncio
+async def test_get_initial_password_action(default_admin_password):
+    """
+    arrange: after WordPress charm has been deployed
+    act: run get-initial-password action
+    assert: get-initial-password action should return default admin password
+    """
+    assert (
```

Comment:
Do you think the action should be triggered from within the test?

Reply:
Fixed, thanks!